composer.json

```diff
@@ -32,24 +32,23 @@
         }
     },
     "require": {
-        "php": "^7.4 || ~8.0.0 || ~8.1.0",
+        "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
         "laminas/laminas-eventmanager": "^3.5",
         "laminas/laminas-servicemanager": "^3.15.1",
         "laminas/laminas-stdlib": "^3.10.1"
     },
     "require-dev": {
-        "laminas/laminas-cache": "^3.1.3",
-        "laminas/laminas-cache-storage-adapter-memory": "^2.0.0",
+        "laminas/laminas-cache": "^3.8",
+        "laminas/laminas-cache-storage-adapter-memory": "^2.2",
         "laminas/laminas-coding-standard": "~2.4.0",
-        "laminas/laminas-db": "^2.13.4",
-        "laminas/laminas-http": "^2.15",
-        "laminas/laminas-validator": "^2.15",
-        "mongodb/mongodb": "~1.13.1",
-        "php-mock/php-mock-phpunit": "^1.1.2 || ^2.0",
-        "phpspec/prophecy-phpunit": "^2.0.1",
+        "laminas/laminas-db": "^2.15",
+        "laminas/laminas-http": "^2.17.1",
+        "laminas/laminas-validator": "^2.28",
+        "mongodb/mongodb": "~1.13.0",
+        "php-mock/php-mock-phpunit": "^1.1.2 || ^2.6.1",
```

Comment:
The trait for `PHPMock` is imported in `SessionConfigTest` but remains unused, so this dependency could be dropped

Reply:
done